x-pack/filebeat/module/cisco/ftd/test/no-type-id.log-expected.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+        "@timestamp": "2018-01-11T01:00:27.000Z",
+        "cisco.ftd.message_id": "430001",
+        "cisco.ftd.security.ApplicationProtocol": "http",
+        "cisco.ftd.security.Client": "webserver",
+        "cisco.ftd.security.DstIP": "10.8.12.47",
+        "cisco.ftd.security.Message": "Intrusion attempt",
+        "cisco.ftd.security.SrcIP": "10.1.123.45",
+        "destination.ip": "10.8.12.47",
+        "event.action": "intrusion-detected",
+        "event.code": "430001",
+        "event.dataset": "cisco.ftd",
+        "event.module": "cisco",
+        "event.severity": 7,
+        "event.timezone": "+00:00",
+        "fileset.name": "ftd",
+        "host.hostname": "beats",
+        "input.type": "log",
+        "log.level": "debug",
+        "log.offset": 0,
+        "log.original": "ApplicationProtocol: http, Client: webserver, DstIP: 10.8.12.47, SrcIP: 10.1.123.45, Message: Intrusion attempt",
```

Comment:
Renamed, but will keep log.original for 7.x to avoid introducing a breaking change in the ASA fileset.

Reply:
The problem is that this FTD module builds on top of the ASA module which came out in 7.2. Both FTD and ASA share the same pipeline because FTD logs are just ASA logs with 5 extra message types.